roles/common/tasks/main.yml

```diff
@@ -1,111 +1,90 @@
 ---
-- name: common | Confirm that hostname matches inventory
+- name: Common | Confirm that hostname matches inventory
   ansible.builtin.hostname:
     name: "{{ inventory_hostname }}"
   when:
     "ansible_hostname not in inventory_hostname"
   tags: "update_hostname"
-- name: common | Add apt HTTPS capabilities.
-  ansible.builtin.package:
-    name: "{{ item }}"
-    state: present
-  loop:
-    - ca-certificates
-    - apt-transport-https
-  when:
-    - ansible_os_family == "Debian"
 
-- name: common | install snapd
-  ansible.builtin.package:
-    name: "{{ item }}"
-    state: present
+- name: Common | Install Common packages
+  ansible.builtin.include_tasks: "{{ item }}"
```

Comment:
We don't need to put these tasks in separate files and include them. The conditions will ensure that the correct packages get installed on the correct OS.